pkg/controller/scheduledjob/controller_test.go

```diff
@@ -90,22 +90,38 @@ func scheduledJob() batch.ScheduledJob {
 					Labels:      map[string]string{"a": "b"},
 					Annotations: map[string]string{"x": "y"},
 				},
-				Spec: batch.JobSpec{
-					Template: api.PodTemplateSpec{
-						ObjectMeta: api.ObjectMeta{
-							Labels: map[string]string{
-								"foo": "bar",
-							},
-						},
-						Spec: api.PodSpec{
-							Containers: []api.Container{
-								{Image: "foo/bar"},
-							},
-						},
-					},
+				Spec: jobSpec(),
+			},
+		},
+	}
+}
+
+func jobSpec() batch.JobSpec {
+	return batch.JobSpec{
+		Template: api.PodTemplateSpec{
+			ObjectMeta: api.ObjectMeta{
+				Labels: map[string]string{
+					"foo": "bar",
 				},
 			},
+			Spec: api.PodSpec{
+				Containers: []api.Container{
+					{Image: "foo/bar"},
+				},
+			},
+		},
+	}
+}
+
+func job(UID string) batch.Job {
+	return batch.Job{
+		ObjectMeta: api.ObjectMeta{
+			UID:       types.UID(UID),
+			Name:      "foobar",
+			Namespace: api.NamespaceDefault,
+			SelfLink:  "/apis/batch/v1/namespaces/snazzycats/jobs/myjob",
```

Comment:
Can't remove it since it's used to reference job (removing it caused `failed to get job's ref: selfLink was empty, can't make reference` error).